api/Window.json

```diff
@@ -9176,7 +9176,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/storage_event",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": true
```

Comment:
This should be "`", chrome_android should be "18" and webview_android should be "≤37". (@Elchi3 should this be in quotation marks?)

Reply:
> chrome_android should be "18" and webview_android should be "≤37"

Just curious, what sources do you use get such detailed information?

Reply:
Except in rare cases, Android always matches Chrome for anything new. The first version of Android Chrome was 18 so anything in Chrome before that is 18 by default. The next version of Android was 25. So anything that landed in Chrome 19 through 24 is Android Chrome 25. The exception are marked on feature cards on [ChromeStatus.com](https://www.chromestatus.com/features). If it's older than the last few years and it says something isn't supported in one of Chrome, Android, or WebView you should send me a message about it. 

For WebView, anything that landed in Chrome from 37 onward will have the same version number as Chrome unless it is in [this list](https://cs.chromium.org/chromium/src/android_webview/tools/system_webview_shell/test/data/webexposed/not-webview-exposed.txt) because it is a Chromium fork. Early versions of Android had a different WebView. These early WebViews are so old, there is currently no practical way to verify what was in them. So if something landed in Chromium before the Chromium WebView we have no way of knowing if it was in both WebViews. The solution we arrived at after months of thought and debate was to mark these items "≤37".

By the way, this value needs to be quoted. There's no need to wait on @Elchi3 's response.